cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/bit_vector.cpp

```diff
@@ -0,0 +1,463 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/algo/subgraph_isomorphism/backend/cpu/bit_vector.hpp"
+
+namespace oneapi::dal::preview::subgraph_isomorphism::backend {
+
+void or_equal(std::uint8_t* ONEAPI_RESTRICT vec,
+              const std::uint8_t* ONEAPI_RESTRICT pa,
+              std::int64_t size) {
+    ONEDAL_IVDEP
+    for (std::int64_t i = 0; i < size; i++) {
+        vec[i] |= pa[i];
+    }
+}
+void and_equal(std::uint8_t* ONEAPI_RESTRICT vec,
```

Comment:
The same question for each part of backend without CPU. Illegal instruction and optimization dependency  may appear (from machine where library is built at).